opendbc/car/byd/bydcan.py

```diff
@@ -0,0 +1,87 @@
+def create_can_steer_command(packer, steer_angle, steer_req, is_standstill, ecu_fault, recovery_btn):
+
+  set_me_xe = 0xE if is_standstill else 0xB
+  eps_ok = not steer_req
+  if recovery_btn:
+    eps_ok = ecu_fault
+
+  values = {
+    "STEER_REQ": steer_req,
+    # to recover from ecu fault, it must be momentarily pulled low.
+    "EPS_OK": eps_ok,
+    "STEER_ANGLE": steer_angle,
+    # must be 0x1 to steer
+    "SET_ME_X01": 0x1 if steer_req else 0,
+    # 0xB fault lesser, maybe higher value fault lesser, 0xB also seem to have the highest angle limit at high speed.
+    "SET_ME_XE": set_me_xe if steer_req else 0,
```

Comment:
we probably don't understand this signal. I'm guessing this is combing a few different signals into one